### PR TITLE
fix: fix the glow effect, and remove whats-new page

### LINKED ISSUE #3: Not working well in VSCode version 1.64
Hi, the glow effect do not work, could you update it? 

And I found issue from [synthwave-vscode #252](https://github.com/robb0wen/synthwave-vscode/issues/252#issuecomment-1031790561).

### PR DESCRIPTION
**Pull request by liray**
Fixes #3

### CODE REVIEW

**Comment by DanielSere** [comment]
> Fixes #3
in **extension.js** you can just comment **showUpdatePage**

**Comment by oasaleh** [comment]
How do I use your updated branch in VSCode?

**Comment by mazwrld** [comment]
Don't you just merge this main branch?

File: src/extension.js [hunk 1]
```diff
@@ -9,28 +9,6 @@ const diff = require('semver/functions/diff');
 function activate(context) {
 	this.extensionName = "AaronYoung.dark-synthwave-vscode";
 	this.cntx = context;
-	this.extension = vscode.extensions.getExtension(this.extensionName);
```
Comment (mazwrld) [inline src/extension.js:12]: So does removing this block of code fix the issue of a new tab?

File: src/extension.js [hunk 2]
```diff
@@ -146,32 +124,12 @@ function activate(context) {
...
-  );
 
 	context.subscriptions.push(disposable);
 	context.subscriptions.push(disable);
-	context.subscriptions.push(whatsNew);
 }
 exports.activate = activate;
 
-
-function showUpdatePage() {
```
Comment (mazwrld) [inline src/extension.js:161]: What if this feature is needed though? Is there no other to get arrount this bug?

**Comment by Rdoolz51** [comment]
if the issue is around the glow effect, can we at least get the whats-new page issue fixed so i can reinstall this extension? thanks

**Comment by brainomite** [comment]
@liray  Thanks and apologies for the long delay in reacting.

**Comment by mazwrld** [comment]
Were you able to figure out how to release the update?

**Comment by brainomite** [comment]
Yes, I made one of my own changes, line number color, it's been published and is updated on [VSC's site](https://marketplace.visualstudio.com/items?itemName=aaronYoung.dark-synthwave-vscode) 
The new version is 1.0.1 and we just need to wait until it makes it through MSFT's system to update our local installs.

**Pull request merged**